draft/2022-02-02-this-week-in-rust.md

```diff
@@ -15,41 +15,60 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ## Updates from Rust Community
 
 ### Official
+* [Changes in the Core Team](https://blog.rust-lang.org/2022/01/31/changes-in-the-core-team.html)
```

Comment:
Might want to add the corresponding changes to the Rust Foundation board along with it. https://foundation.rust-lang.org/posts/2022-02-02-farewell-florian/

Reply:
I'll add that in :)

Reply:
Done!